src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -1025,22 +1027,45 @@ private static SeatBid toSeatBid(BidderResponse bidderResponse, TargetingKeyword
     /**
      * Returns an OpenRTB {@link Bid} with "prebid" and "bidder" extension fields populated.
      */
-    private static Bid toBid(BidderBid bidderBid, String bidder, TargetingKeywordsCreator keywordsCreator,
+    private Bid toBid(BidderBid bidderBid, String bidder, TargetingKeywordsCreator keywordsCreator,
                              Map<Bid, CacheIdInfo> winningBidsWithCacheIds, Set<Bid> winningBidsByBidder) {
         final Bid bid = bidderBid.getBid();
 
         final Map<String, String> targetingKeywords;
+        Cache.CacheBuilder cacheBuilder = null;
```

Comment:
Use these like variables in lower scope, please.
i mean in `if (keywordsCreator != null && winningBidsByBidder.contains(bid)) {...}`.